packages/elements/src/components/ino-popover/ino-popover.tsx

```diff
@@ -72,44 +81,111 @@ export class Popover implements ComponentInterface {
    */
   @Method()
   async getTippyInstance(): Promise<any> {
-    return this.tooltipInstance;
+    return this.tippyInstance;
   }
 
-  // Lifecycle
+  /**
+   * Used to indicate if the popover should be controlled by itself (`false`) or manually by the `ino-show` property (`true`)
+   */
+  @Prop() inoControlled: boolean = false;
 
-  componentDidLoad() {
+  @Watch('inoControlled')
+  inoControlledChanged() {
     this.create();
   }
 
-  // Private methods
+  /**
+   * Programmatically show or hide the popover.
```

Comment:
Show or hide the popover.

In general I think the naming (`inoShow`) and (`visibilityChanged`) is not perfect.

What do you think of `inoVisible` and `inoVisibleChanged`?